operations/spackle.js

```diff
@@ -5,7 +5,61 @@ var fs = require('fs');
 var childProcess = require('child_process');
 var deltas = require('./deltas');
 
-var writtenFiles = [5, 2015, 2016, 2017, 2018, 2019].flatMap((year, i, arr) => {
+var GetIntrinsic = require('../GetIntrinsic');
+var callBound = require('../helpers/callBound');
+var callBind = require('../helpers/callBind');
+
+/** @type {() => never} */
+var $ThrowTypeError = GetIntrinsic('%ThrowTypeError%');
+
+/** @type {(arg: any) => string} */
+var $ObjProtoToString = callBound('%Object.prototype.toString%');
+
+/** @type {(arg: any) => arg is (any[] | ReadonlyArray<any>)} */
+var $isArray = GetIntrinsic('%Array.isArray%', true)
+	|| function (arg) {
+		return $ObjProtoToString(arg) === '[object Array]';
+	};
+
+/** @type {<T>(array: T[], ...items: T[]) => number} */
```

Comment:
Please do not add typescript-like comments to this repo; I’m not interested in the pollution.